.github/workflows/nightly.yml

```diff
@@ -34,6 +34,8 @@ jobs:
           HOMEBREW_NO_INSTALL_FROM_API: 1
         run: brew install semgrep --HEAD
       - name: Check installed correctly
+        env:
+          HOMEBREW_NO_INSTALL_FROM_API: 1
```

Comment:
Please add a comment explaining this magic constant.
A ref to a github issue explaining the need for this kind of fix, whatever, so in a few months we remember
what it was useful for.